### DISCUSSION TITLE
Base Theme capabilities

### DISCUSSION THREAD

**Discussion by eznix86** [founded-labs/react-native-reusables#15] [Ideas]
Providing a base theme capabilities could help users just copy and paste what they need.

Similar to https://ui.shadcn.com/themes. We can provide already built-in themes, with specific colors and shapes.

One proposal for built-in themes could be:
- The base one (what we already have),
- Then something closer to Android Ecosystem (Material 2 or Material You) 
- and iOS Ecosystem (Human Design Interface) in term of colors and shape.

I acknowledge that the current design system is not ever close to those ecosystem, but we can have base colors and shapes which can be closer to those Android and iOS.

Currently we can override global.css, we can use that as a base for the default, material and ios feel then the rest can be just colors and shapes.

**Comment by mrzachnugent** [comment]
If you want inspiration for your own theme, check out: https://ui.jln.dev/. And make sure you use .dark:root as the selector in your css file instead of .dark
